### ISSUE TITLE
Your model has data leakage issues. 

### ISSUE THREAD

**Issue by ittanweiming** [cherishdd/VDBFL#1]
Your model has data leakage issues. Although only the training set was trained during the feature extraction phase, the training set, validation set, and test set were mixed during the detection phase
